examples/layout-analysis/visualization.py

```diff
@@ -0,0 +1,80 @@
+import os
+import pathlib
+import sys
+
+import pdf2image
+from unstructured_inference.inference.elements import Rectangle
+from unstructured_inference.visualize import draw_bbox
+
+from unstructured.documents.elements import PageBreak
+from unstructured.partition.pdf import partition_pdf
+
+CUR_DIR = pathlib.Path(__file__).parent.resolve()
+
+
+def extract_element_coordinates(elements):
+    elements_coordinates = []
+    page_elements_coordinates = []
+
+    for el in elements:
+        if isinstance(el, PageBreak):
+            if page_elements_coordinates:
+                elements_coordinates.append(page_elements_coordinates)
+                page_elements_coordinates = []
+        else:
+            page_elements_coordinates.append(el.metadata.coordinates)
+
+    if page_elements_coordinates:
+        elements_coordinates.append(page_elements_coordinates)
+
+    return elements_coordinates
+
+
+def run_partition_pdf(f_path, strategy, images, output_dir):
+
+    elements = partition_pdf(
+        f_path,
+        strategy=strategy,
+        include_page_breaks=True,
+    )
+
+    elements_coordinates = extract_element_coordinates(elements)
+    assert len(images) == len(elements_coordinates)
+
+    for idx, (img, coords_per_page) in enumerate(zip(images, elements_coordinates)):
+        for coordinate in coords_per_page:
+            points = coordinate.points
+            x1, y1 = points[0]
+            x2, y2 = points[2]
+            rect = Rectangle(x1, y1, x2, y2)
+            img = draw_bbox(img, rect, color="red")
+
+        output_image_path = os.path.join(output_dir, f"{strategy}-{idx + 1}.jpg")
```

Comment:
please print the path here for convenience

Reply:
Sounds good!